templates/modals.html

```diff
@@ -46,7 +46,7 @@ <h5 class="modal-title" id="inputModalTitle">Title</h5>
 </div>
 
 <!-- Request User Text Input Modal -->
-<div class="modal fade" id="inputTextModal" tabindex="-1" role="dialog" aria-labelledby="inputTextModalTitle" aria-hidden="true">
```

Comment:
The `aria-hidden="true"` parameter seems to be used in the example modals in the bootstrap docs:

https://getbootstrap.com/docs/5.3/components/modal/#live-demo

Having said that, I don't know what they do? What error did they cause than meant they had to be removed?

Reply:
yea i saw they were being used, but in practise for eagle, removing them seemed to  change nothing. the error popped up seemingly randomly. it was just a console error, no issues but i figured since they didnt seem to do anything ill just remove them.

Reply:
I looked it up, it removes the element from the accessibility tree. So only useful for screen readers etc. I'm not sure why bootstrap hides modals from accessibility, surely they would need to be read by a screen reader.

Anyway, it should no be a problem to remove the aria-hidden params.